deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -39,3 +42,37 @@ def callback_detect_vulnerability_detector_enabled(line):
     match2 = re.match(msg, line)
 
     return match1 is not None and match2 is None
+
+
+def make_callback(pattern):
+    """
+    Creates a callback function from a text pattern.
+
+    It already contains the vulnerability-detector prefix.
+
+    Parameters
+    ----------
+    pattern : str
+        String to match on the log
+
+    >>> callback_bionic_update_started = make_callback("Starting Ubuntu Bionic database update")
+    """
+    pattern = r'\s+'.join(pattern.split())
+    msg = modulesd_prefix + pattern
+    regex = re.compile(msg)
+
+    def _callback(line):
+        match = regex.match(line)
+        return match is not None
+    return _callback
+
+
+def callback_detect_no_feeds_downloaded(line):
+    """This tests that no feeds are being downloaded"""
+    update_regex = re.compile(modulesd_prefix + 'Starting.*database\s+update')
+    scan_regex = re.compile(modulesd_prefix + 'Starting\s+vulnerability\s+scanning')
+    update = update_regex.match(line)
+    if update:
+        raise Exception("A feed download was detected")
```

Comment:
```suggestion
        raise AttributeError("A feed download was detected")
```

For consistency with other tests.

Reply:
This function was deprecated in favor on this approach: https://github.com/wazuh/wazuh-qa/pull/689/files#diff-d60a2e08b9d1f2d1f836bec1cd1176f3R65